cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -458,14 +479,14 @@ public UserTaskInfo(HttpServletRequest httpServletRequest,
            httpServletRequest.getParameterMap(), ParameterUtils.endPoint(httpServletRequest), state);
     }
 
-    public UserTaskInfo(List<OperationFuture> futures,
-                        String requestUrl,
-                        String clientIdentity,
-                        long startMs,
-                        UUID userTaskId,
-                        Map<String, String[]> queryParams,
-                        EndPoint endPoint,
-                        TaskState state) {
+    UserTaskInfo(List<OperationFuture> futures,
```

Comment:
Would you clarify why you make `UserTaskInfo` and its methods `package-private`?